src/backend/utils/github-url.js

```diff
@@ -56,69 +56,70 @@ exports.getGithubUrlData = async incomingUrl => {
     typeof process.env.GITHUB_TOKEN !== 'undefined'
       ? `?access_token=${process.env.GITHUB_TOKEN}`
       : '';
+  try {
+    const fetchResult = await fetch(`${githubAPI}${subUrl}`);
+    const data = await fetchResult.json();
```

Comment:
Put your `catch` here vs. at the bottom, it's only these 2 lines that can throw.